packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -301,6 +305,14 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
 
     return false;
   }
+  /**
+   *  update the forceAlignemnt state with true/false value.
+   *  @param forceAlignemntState - if true ,focus Alignemnt will be recalculated depending on the currenet active element position.
```

Comment:
typo - can we make this `forceAlignmentState`

Reply:
sure :)